cli_tools/gce_image_publish/publish/publish.go

```diff
@@ -473,22 +491,36 @@ func (p *Publish) deprecatePrintOut(deprecateImages *daisy.DeprecateImages) {
 
 	for _, di := range *deprecateImages {
 		image := path.Base(di.Image)
-		switch di.DeprecationStatus.State {
+		switch di.DeprecationStatusAlpha.State {
 		case "DEPRECATED":
 			p.toDeprecate = append(p.toDeprecate, image)
 		case "OBSOLETE":
 			p.toObsolete = append(p.toObsolete, image)
-		case "":
+		case "ACTIVE", "":
 			p.toUndeprecate = append(p.toUndeprecate, image)
 		}
 	}
 }
 
-func (p *Publish) populateWorkflow(ctx context.Context, w *daisy.Workflow, pubImgs []*compute.Image, img *Image, rb, sd, rep, noRoot bool) error {
+func (p *Publish) rolloutPolicyPrintOut(rp *computeAlpha.RolloutPolicy) {
```

Comment:
This function is named like it's printing rollout policies but it's actually building a printout. Consider either renaming it to something like `buildRolloutPolicyPrintOut` or change the behavior if this is an oversight.